vyper/semantics/types/user.py

```diff
@@ -381,38 +381,37 @@ def to_abi_arg(self, name: str = "") -> dict:
         components = [t.to_abi_arg(name=k) for k, t in self.member_types.items()]
         return {"name": name, "type": "tuple", "components": components}
 
-    # TODO breaking change: use kwargs instead of dict
-    # when using the type itself (not an instance) in the call position
-    # maybe rename to _ctor_call_return
     def _ctor_call_return(self, node: vy_ast.Call) -> "StructT":
-        validate_call_args(node, 1)
-        if not isinstance(node.args[0], vy_ast.Dict):
+        if len(node.args) > 0:
```

Comment:
is there a test for this?

Reply:
I don't think it's unreachable though, e.g.`MyStruct(a, x=y)`

Reply:
oh ic, I have just added a syntax test.